File: src/java/Fibonacci.java
import java.math.BigInteger;

public class Fibonacci {
    public static void main(String[] args) {
        String countEnv = System.getenv("COUNT");
        int count = 10;
        if (countEnv != null) {
            try {
                count = Integer.parseInt(countEnv);
            } catch (NumberFormatException e) {
            }
        }

        BigInteger a = BigInteger.ZERO;
        BigInteger b = BigInteger.ONE;

        for (int i = 0; i < count; i++) {
            BigInteger temp = a;
            a = b;
            b = b.add(temp);
        }
        System.out.println("Result(F_" + count + "): " + a);
    }
}

File: src/js/fibonacci.js
function fibonacci(n) {
    let a = 0n, b = 1n;
    for (let i = 0; i < n; i++) {
        process.stdout.write(a + " ");
        let temp = a;
        a = b;
        b = temp + b;
    }
}

const count = parseInt(process.env.COUNT || '10');
console.log(`Fibonacci Sequence (first ${count} numbers):`);
fibonacci(count);
